is3106backend/users/views.py

```diff
@@ -230,14 +225,14 @@ def protected_user_delivery_address_view(request):
 
 @api_view(['DELETE'])
 @permission_classes((IsAuthenticated,))
-def protected_user_delivery_address_delete_view(request, pk):
+def protected_user_delivery_address_delete_view(request, pk, jk):
```

Comment:
why is delivery address called jk lol

Reply:
cant be pk, theres not really a reason why I named it jk. Should I change it to dk? dpk?
```